cmd/build.go

```diff
@@ -383,7 +383,7 @@ func ContainerRequirements(abspath string, name string, version string, logger l
 		// create map of regexp patterns to search for in Dockerfile as well as log information if not found
 		m := map[string]string{
 			"FROM quay\\.io/centos/centos:stream8":                                             "Dockerfile doesn't use 'FROM quay.io/centos/centos:stream8'\n",
-			"(ADD|COPY) .*/LICENSE /.*":                                                        "Dockerfile does not contain copy of arcaflow plugin license\n",
+			"(ADD|COPY) LICENSE /.*":                                                           "Dockerfile does not contain copy of arcaflow plugin license\n",
```

Comment:
Do we want to enforce the new way of copying the local file, or do we plan to support both ways? Because if we want to support both ways, adding a ? after the / and keeping the .* should support that. It would make it optional.

Reply:
I have updated